Essentials/src/plugin.yml

```
@@ -216,6 +216,10 @@ commands:
     description: Searches for an item.
     usage: /<command> <item>
     aliases: [dura,edura,durability,edurability,eitemdb,itemno,eitemno]
+  itemlore:
+    description: Edit the lore of an item.
+    usage: /<command> <add/set/clear> [text/line] [text]
+    aliases: [ilore, eilore, eitemlore]
```

```suggestion
    aliases: [lore, elore, ilore, eilore, eitemlore]
```

`/lore` seems like a reasonable alias - whereas `/name` for `/itemname` might be ambiguous, `/lore` seems to be more obviously related to items.